_config.yml

```diff
@@ -111,6 +111,12 @@ css_inline: false # Will insert all styles into a single <style> block in the <h
 navigation_header:
 - title: Home
   url: /
+- title: Documentation
+  url: https://www.juliaopt.org/JuMP.jl/stable/
+- title: Forum
+  url: https://discourse.julialang.org/c/domain/opt
+- title: Code
+  url: https://github.com/juliaopt/JuMP.jl
```

Comment:
Maybe this should point to a page that explains how JuMP's code (if you include solver wrappers) is structured across ~20 repositories?